From 89e78a8033f79e24480012fe646973bc5fce18e0 Mon Sep 17 00:00:00 2001
From: Jason Yu <xuzj03@gmail.com>
Date: Fri, 30 Nov 2018 00:38:40 +0800
Subject: [PATCH] Add newlib-nano build support

---
 Makefile.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Makefile.in b/Makefile.in
index 8873ea8..553cf6e 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -257,6 +257,8 @@ stamps/build-newlib: $(srcdir)/riscv-newlib stamps/build-gcc-newlib-stage1
 		--enable-newlib-io-long-double \
 		--enable-newlib-io-long-long \
 		--enable-newlib-io-c99-formats \
+        --enable-newlib-nano-malloc \
+        --enable-newlib-nano-formatted-io \
 		CFLAGS_FOR_TARGET="-Os $(CFLAGS_FOR_TARGET)"
 	$(MAKE) -C $(notdir $@)
 	$(MAKE) -C $(notdir $@) install
-- 
2.17.0

